modules/ui/UiOvertureInspector.js

```diff
@@ -0,0 +1,302 @@
+ * Because Overture entities conform to a certain schema, we might at some point build a JSON-Schema-aware 
+ * version of this code that modifies the display of the data. 
+ *
+ * @example
+ *  <div class='overture-inspector'>
+ *    <div class='header'>…</div>
+ *    <div class='body'>
+ *      <div class='theme-info'/>              // Theme name, e.g. "Places" or "Addresses"
+ *      <div class='property-info'/>           // List of properties on this feature
+ *    </div>
+ *  </div>
+ */
+export class UiOvertureInspector {
+  /**
+   * @constructor
+   * @param  `context`  Global shared application context
+   */
+  constructor(context) {
+    this.context = context;
+
+    this.datum = null;
+    this._keys = null;
+
+    // D3 selections
+    this.$parent = null;
+    this.$inspector = null;
+
+    // Create child components
+    this.AcceptTooltip = uiTooltip(context).placement('bottom');
+    this.IgnoreTooltip = uiTooltip(context).placement('bottom');
+
+    // Ensure methods used as callbacks always have `this` bound correctly.
+    // (This is also necessary when using `d3-selection.call`)
+    this.render = this.render.bind(this);
+    this.renderFeatureInfo = this.renderFeatureInfo.bind(this);
+    this.renderPropertyInfo = this.renderPropertyInfo.bind(this);
+    this.renderNotice = this.renderNotice.bind(this);
+  }
+
+
+  /**
+   * render
+   * Accepts a parent selection, and renders the content under it.
+   * (The parent selection is required the first time, but can be inferred on subsequent renders.)
+   * @param {d3-selection} $parent - A d3-selection to a HTMLElement that this component should render itself into
+   */
+  render($parent = this.$parent) {
+    if ($parent instanceof selection) {
+      this.$parent = $parent;
+    } else {
+      return;   // no parent - called too early?
+    }
+
+    const context = this.context;
+    const l10n = context.systems.l10n;
+    const rtl = l10n.isRTL() ? '-rtl' : '';
+
+    let $inspector = $parent.selectAll('.overture-inspector')
+      .data([0]);
+
+    const $$inspector = $inspector.enter()
+      .append('div')
+      .attr('class', 'overture-inspector');
+
+
+    // add `.header`
+    const $$header = $$inspector
+      .append('div')
+      .attr('class', 'header');
+
+    $$header
+      .append('h3')
+      .append('img')
+      .attr('class', 'wordmark-overture')
+      .attr('src', this.context.assetPath + 'img/omf-wordmark.svg');
+
+    $$header
+      .append('button')
+      .attr('class', 'overture-inspector-close')
+      .on('click', () => context.enter('browse'))
+      .call(uiIcon('#overture-icon-close'));
+
+    // add `.body`
+    $$inspector
+      .append('div')
+      .attr('class', 'body');
+
+    // update
+    this.$inspector = $inspector = $inspector.merge($$inspector);
+
+    // localize logo
+    $inspector.selectAll('.logo-overture > use')
+      .attr('xlink:href', `#overture-logo-overture-wordmark${rtl}`);
```

Comment:
It looks like this code needs to change a little bit for right-to-left.. This is optional and shouldn't hold up the PR.
We could remove the `'src'` attribute up on line 87 (in enter selection), and put some code here that sets the `'src'` to use either the normal or  rtl version of the logo (in update selection).  
We'd need to actually make that logo too, but that's easy.

Reply:
Ah yes! I meant to ask you about the RTL code you added.  That's fairly new to me, and yeah the wordmark is available in one flavor. I'm .... actually not sure how brands handle wordmarks like this. Is it okay to flip them without permission from the entity that owns it? Hmmmm.

Reply:
Donezo! Please see screenies of the ltr-vs-rtl in the original updated description.